Makefile

```diff
@@ -26,34 +29,51 @@
 # run all checks (including tests, dialyze and gradualize)
 # > make check
 #
-PROJECT = gradualizer
 
-PLT_APPS = kernel stdlib compiler crypto
-DIALYZER_OPTS = -Werror_handling
+.PHONY: app
+app:
+	rebar3 compile
 
-ESCRIPT_NAME = gradualizer_cli
-ESCRIPT_FILE = $(PROJECT)
+.PHONY: shell
+shell:
+	rebar3 shell
 
-include erlang.mk
+.PHONY: escript
+escript:
+	rebar3 escriptize
+	cp _build/default/bin/gradualizer .
 
+.PHONY: gradualize
 gradualize: escript
 	./gradualizer -pa $(CURDIR)/ebin/ $(CURDIR)/ebin/*beam
-check:: gradualize
 
-# We want warnings to be warnings, not errors.
-ERLC_OPTS := $(filter-out -Werror,$(ERLC_OPTS))
+.PHONY: clean
+clean:
+	rebar3 clean
+	rm -rf _build
+	rm .gradualizer.plt
+	rm gradualizer
+
+.PHONY: tests
+tests:
+	rebar3 eunit
+
+.PHONY: cover
+cover:
+	rebar3 do eunit -c, cover
 
-EUNIT_OPTS = verbose
+.PHONY: showcover
+showcover: cover
+	open _build/test/cover/index.html
 
-# Unit test files to run
-EUNIT_TEST_SOURCES = $(shell ls $(TEST_DIR)/*.erl)
+.PHONY: dialyze
```

Comment:
this `dialyze`/`gradualize` was just an `erlang.mk` naming convention. let's just use `dialyzer`/`gradualizer targets.

Reply:
We can't keep both around? 😸😈
Or perhaps we could use `dialyze` in the make file and those that want to use `dialyzer` can use `rebar3 dialyzer`?
**Important stuff!**

Reply:
I like the suggestions of using `gradualize` in the makefile and `gradualizer` in rebar3. Having the make target `gradualizer` to run the gradualizer on itself is rather confusing because its name suggest that it would build the command line program "gradualizer".

Reply:
What about `dialyze`?

Reply:
Yes, I like `dialyze` too! We should make sure to use the same naming convention for both targets.

Reply:
let's keep the old `gradualize`/`dialyze` that we all got used to and love in the make file then.